mdLinks.js

```diff
@@ -0,0 +1,100 @@
+//console.log("hola mundo")
+
+const fs = require("fs")
+const fetch = require('node-fetch');
+//const readLine = require ("readline").createInterface({
+    //input : process.stdin,
+    //output : process.stdout
+//})
+//const check = require ("check-broken-links")
+//const hljs   = require('highlight.js');
+//const marked = require('marked');
+
+let data = fs.readFileSync("new.md","utf-8");
+
+
+function urlify(data) {
+    let urlRegex = /(https?:\/\/[^\s)]+)/gi;
+    return data.match(urlRegex)
+   // console.log(url.length)
+    // or alternatively
+    // return text.replace(urlRegex, '<a href="$1">$1</a>')
+}
+console.log(urlify(data).length);
+console.log(urlify(data));
+let work = 0;
+    let broke = 0;
+
+function isValidURL(){
```

Comment:
me gusta que ya validas los links y aun sigues teniendo poco código, me confundí un poco en el for y pero creo que al final termine por entender que es lo que hace